server/graphql/v2/object/AccountStats.js

```diff
@@ -80,6 +82,36 @@ export const AccountStats = new GraphQLObjectType({
           return collective.getTotalAmountReceivedAmount({ kind, startDate: args.dateFrom, endDate: args.dateTo });
         },
       },
+      totalPaidExpense: {
+        description: 'Total of paid expenses, filter per expensetype',
+        type: new GraphQLNonNull(Amount),
+        args: {
+          expenseType: {
+            type: new GraphQLList(ExpenseType),
+            description: 'Filter by ExpenseType',
+          },
+          dateTo: {
+            type: GraphQLDateTime,
+            description: 'Calculate total amount received before this date',
+          },
+          dateFrom: {
+            type: GraphQLDateTime,
+            description: 'Calculate total amount received after this date',
+          },
+          currency: {
+            type: GraphQLString,
+            description: 'An optional currency. If not provided, will use the collective currency.',
+          },
+        },
+        async resolve(collective, args) {
+          return await collective.getTotalAmountPaidExpensesAmount({
```

Comment:
`return await` can be just `return`.

Reply:
`getTotalAmountPaidExpensesAmount` is a weird name with 2 times "amount" in it. Update with something else?

Reply:
I actually see that "getTotalAmountReceivedAmount" is already existing so that's fine ... We should do better later.

Reply:
In the end, whatever the other, `getTotalPaidExpensesAmount` looks clearer.